suite/suite.go

```diff
@@ -84,7 +84,7 @@ func Run(t *testing.T, suite TestingSuite) {
 	defer failOnPanic(t)
 
 	suiteSetupDone := false
-	
+
```

Comment:
These changes are required because TravisCI is running the gofmt check again. A couple files weren't formatted properly.